themes/bootstrap3/templates/search/pagination.phtml

```diff
@@ -1,16 +1,21 @@
 <?php if ($this->pageCount): ?>
   <?php
     $showFirstLast = $this->pageCount > 3 && $this->results->getOptions()->firstLastNavigationSupported();
+    $navAttrs = $this->htmlAttributes(['aria-label' => $this->translate('pagination_label')]);
+    if ($this->results->getOptions()->loadResultsWithJsEnabled()) {
+      $navAttrs->add('class', 'js-pagination');
+    }
+    $className = $this->options['className'] ?? '';
+    if ($className) {
+      $navAttrs->add('class', $className);
```

Comment:
Is it intentional that you are adding the same class to both the nav and the ul when the className option is set? If so, maybe it's worth a comment explaining this so we don't mistake it for a bug in future.

Reply:
Yes it is. It could probably be removed from the `ul` element since I left it there for BC compatibility only. Or I'll just revise it so that there's a different attribute for the nav class.

Reply:
Actually, I'll revise the whole mechanism a bit. :)

Reply:
Now revised so that there doesn't need to be so much logic outside.